PYME/Acquire/Scripts/init_UOA_n.py

```diff
@@ -235,7 +235,7 @@ def drift_tracking(MainFrame, scope):
 
     def _drift_init():
         #scope.p_drift = subprocess.Popen('%s "%s" -i init_drift_tracking.py -t "Drift Tracking" -m "compact"' % (sys.executable, PYMEAcquire.__file__), shell=True)
-        scope.p_drift = subprocess.Popen('%s "%s" -i init_drift_tracking.py -t "Drift Tracking"' % (sys.executable, PYMEAcquire.__file__), shell=True)
+        scope.p_drift = subprocess.Popen('%s "%s" -i init_drift_tracking.py -t "Drift Tracking"' % (sys.executable, PYMEAcquire.__file__), creationflags=subprocess.CREATE_NEW_CONSOLE)
```

Comment:
the docs for the subprocess module state that `shell=True` implies `creationflags=subprocess.CREATE_NEW_CONSOLE` is that not the case? Does setting `creationflags=subprocess.CREATE_NEW_CONSOLE` actually create a new window?

Reply:
yes. 
![image](https://user-images.githubusercontent.com/31105780/94492843-2ff38c80-01b9-11eb-83ac-e5b4bec1fc7e.png)

Reply:
Also, I'm not seeing anything in the 3.6 docs on shell=True implying CREATE_NEW_CONSOLE

Reply:
This is specifically an init file for a py2 system, but I suspect the docs are not correct. In any case we should run into this problem:

"If passing a single string, either shell must be True (see below) or else the string must simply name the program to be executed without specifying any arguments" (from 3.7 docs, but reasonably consistent across versions). Will play when I get time, but reverting back here for now.